lecture/views.py

```
@@ -0,0 +1,56 @@
+def lecture_home(request: HttpRequest) -> HttpResponse:
+    """View to show Lecture details."""
+    
+    slug = request.GET.get('slug')  # Get the slug from the query parameter
+
+    
+    classroom = get_object_or_404(Classroom, slug=slug)
+    lectures = Lecture.objects.filter(classroom=classroom).prefetch_related('pdf_files')
+
+    return render(request, 'lecture/index.html', {
+        'classroom': classroom,
+        'lectures': lectures
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)